python/lsst/ts/phosim/utils/ConvertZernikesToPsfWidth.py

```diff
@@ -0,0 +1,132 @@
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import galsim
+import numpy as np
+from lsst.ts.wep.cwfs.Instrument import Instrument
+from lsst.ts.wep.Utility import CamType
+
+
+def getPsfGradPerZernike(
+    jmax: int = 22,
+    inst: Instrument = None,
+) -> np.ndarray:
+    """Get the gradient of the PSF FWHM with respect to each Zernike.
+
+    Parameters
+    ----------
+    jmax : int
+        The max Zernike Noll index, inclusive. (the default is 22.)
+    inst : Instrument, optional
+        The ts.wep instrument used in the Algorithm class to solve the TIE.
+        (the default is the default LsstCam.)
+
+    Returns
+    -------
+    np.ndarray
+        Gradient of the PSF FWHM with respect to the corresponding Zernike.
+        Units are arcsec / micron.
+    """
+    # If inst is None, get the default LsstCame
+    if inst is None:
+        inst = Instrument()
+        inst.configFromFile(
+            160,  # Donut dimension; irrelevant for this function
+            CamType.LsstCam,  # Default is LsstCam
+        )
+
+    # Get the inner and outer radii of the instrument
+    R_outer = inst.apertureDiameter / 2
+    R_inner = R_outer * inst.obscuration
+
+    # Calculate the conversion factors
+    conversion_factors = np.zeros(jmax + 1)
+    for i in range(4, jmax + 1):
+        # Set coefficients for this Noll index: coefs = [0, 0, ..., 1]
+        # Note the first coefficient is Noll index 0, which does not exist and
+        # is therefore always ignored by galsim
+        coefs = [0] * i + [1]
+
+        # Create the Zernike polynomial with these coefficients
+        Z = galsim.zernike.Zernike(coefs, R_outer=R_outer, R_inner=R_inner)
+
+        # We can calculate the size of the PSF from the RMS of the gradient of
+        # the wavefront. The gradient of the wavefront perturbs photon paths.
+        # The RMS quantifies the size of the collective perturbation.
+        # If we expand the wavefront gradient in another series of Zernike
+        # polynomials, we can exploit the orthonormality of the Zernikes to
+        # calculate the RMS from the Zernike coefficients.
+        rms_tilt = np.sqrt(np.sum(Z.gradX.coef**2 + Z.gradY.coef**2))
```

Comment:
Did you run `black` in this line of code? I thought it would be `Z.gradX.coef ** 2` by `black`?